nfpm_test.go

```diff
@@ -149,6 +150,46 @@ func TestParseFile(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("v%s", os.Getenv("GOROOT")), config.Version)
 }
 
+func TestOptionsFromEnvironment(t *testing.T) {
+	const (
+		globalPass = "hunter2"
+		debPass    = "password123"
+		rpmPass    = "secret"
+		apkPass    = "foobar"
+		release    = "3"
+		version    = "1.0.0"
+	)
+
+	os.Clearenv()
```

Comment:
seems like several tests in the same test method, maybe make it clearer using `t.Run()`? 🤔

Reply:
Good idea